contrib/docker/solr/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN mkdir -p /opt/solr/server/solr/$SOLR_CORE/data
 
 # Adding Files
 ADD ./solrconfig.xml \
-https://raw.githubusercontent.com/ckan/ckan/ckan-2.5.2/ckan/config/solr/schema.xml \
+https://raw.githubusercontent.com/ckan/ckan/master/ckan/config/solr/schema.xml \
```

Comment:
so how do we fix this? Moving the solr schema under contrib/docker is pretty terrible.

How about using make to build our docker images? make doesn't have silly limitations like this.